editor/modes/visual-editor/index.js

```diff
@@ -73,6 +81,11 @@ class VisualEditor extends Component {
 		event.preventDefault();
 	}
 
+	save( event ) {
```

Comment:
They use a new pattern in Calypso to avoid binding in the constructor:

```js
save = ( event ) => {
    ...
}
```
I think this is also how the official React codemod migrates `React.createClass` to class component.

Reply:
The feature you're referring to ([public class fields property initializers](https://tc39.github.io/proposal-class-public-fields/)) is actually not part of the language specification, but rather a stage 2 proposal. We've tended to avoid the most language proposal features out of fear for churn, and the syntax will fail to build. With class fields specifically there's been some recent revisions to [combine proposals for public and private fields](https://github.com/tc39/proposal-class-fields) and [unified class features with decorators](https://github.com/tc39/proposal-unified-class-features) that gives me cause for concern (introducing uncertainty to their stability).

Reply:
That makes a lot of sense in the context of WordPress development. I totally support this choice 👍  Thanks for taking your time to explain the rationale behind it :)